search-ui/src/App.vue

```diff
@@ -63,4 +70,10 @@ const handleError = (error: ErrorI) => {
   // FUTURE: add account info with error information 
   Sentry.captureException(error)
 }
+
+const breadcrumbs = (): Array<BreadcrumbIF> => {
```

Comment:
you should be checking the route name in here and passing the correct array of breadcrumbs for each one (currently the breadcrumb should only be active when the route is dashboard). In that case you should have 2 items in the array: bc online services (href = REGISTRY_URL) and the dashboard item. In this case the back button should take you to bc online services

Reply:
you mean two params. First one for back button and the other one to display items right?

Reply:
No I mean you should have 2 breadcrumb items: one for bc online services and one for the dashboard